src/i18n/locale/en.json

```diff
@@ -269,10 +269,10 @@
       },
       "sort": {
         "label": {
-          "alphabetically": "Alphabetically",
+          "alphabetically": "A-Z",
           "by_date_added": "By Date Added",
           "by_last_read": "By Last Read",
-          "by_unread_chapters": "By Unread chapters"
+          "by_unread_chapters": "By Unread Chapters"
```

Comment:
proper noun

Reply:
imo that should be all lowercase letters (e.g. "By date added")